### DISCUSSION TITLE
Nextcloud asks for login data on publicly shared folder

### DISCUSSION THREAD

**Discussion by n-aggelidis** [nextcloud/all-in-one#3371] [Questions] [answered]
Hello there!

I just noticed that when I create a public link with full read and ride permissions and via that link try to copy or move a file inside the folder, my browser prompts me to enter login data. When I press cancel and then try to copy, it does not work. Also you can see in the screenshot that the files list in the copy dialog is not loading, even though it is inside the shared folder. I am not sure if this is the intended behavior, i would guess that is not. I cannot say that this is due to an update, but still I am running the latest AIO Version.

Thanks in advance for your help!
![Nextcloud login prompt](https://github.com/nextcloud/all-in-one/assets/49533513/8834f843-721e-4b07-b447-a26c028f0535)

**Comment by szaimen** [comment, marked as answer]
Hi, this needs to be fixed in nextcloud/server
